src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -110,10 +172,16 @@ var Marathon = React.createClass({
       error: function () {
         this.setState({fetchState: States.STATE_ERROR});
       }.bind(this),
-      reset: true,
       success: function () {
+        var state = this.state;
```

Comment:
this.state is used twice, so I can save one object lookup.
In a larger scale this gains performance and readability.